src/components/mutually-exclusive/_macro-options.md

```diff
@@ -1,8 +1,18 @@
-| error                     | `Error` [_(ref)_](/components/error) | false    | Configuration for validation errors                                                                                                                                                                        |
+| Name                             | Type                                 | Required                                     | Description                                                                                                                                                                                                |
+| -------------------------------- | ------------------------------------ | -------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| id                               | string                               | false                                        | The HTML `id` of the fieldset                                                                                                                                                                              |
+| classes                          | string                               | false                                        | Classes to apply to the fieldset                                                                                                                                                                           |
+| legend                           | string                               | true (unless `legendIsQuestionTitle` is set) | Text for the fieldset’s legend                                                                                                                                                                             |
+| legendClasses                    | string                               | false                                        | Classes to apply to the legend element                                                                                                                                                                     |
+| legendIsQuestionTitle            | boolean                              | true (unless `legend` is set)                | Creates an `h1` inside the `legend`. Use when there is only a single fieldset on the page                                                                                                                  |
+| legendClasses                    | string                               | false                                        | Classes to apply to the legend element                                                                                                                                                                     |
```

Comment:
This one needs to be removed because you have it twice

Reply:
oh sry might have copied it by mistake